### ISSUE TITLE
Threshold config option not working 

### ISSUE THREAD

**Issue by Temi89** [mfrachet/cypress-audit#98]
HI there according to PA11y documentation when running in CI there a config to pass so builds dont fail . 

Ive got the following in my cypress script but seems to do nothing . What I am after to to perform the accessibility scan generate the reports but not fail the test if the violations are within  the threshold . Currently with this config set I have 9 violations and the tests are failing instead of passing.

>    cy.pa11y({
>       runners: ['axe'],
>       threshold: 10,
>       standard: 'WCAG2AA'
>     });

Please is there a way we can do this with PA11y integrated with cypress

**Comment by AgentChris** [comment]
It would be nice to have the ability to set the threshold for pa11y

**Comment by mfrachet** [comment]
I've upgraded pa11y to the version 6. Could you try again using cypress-audit v1.1.0?

I've tried debugging, and it looks that the options passed to the raw pa11y library make sense, so I'm not sure what's going on here 🤔
